adoc/deployment-sysreqs.adoc

```diff
@@ -203,13 +203,39 @@ The management workstation needs at least the following networking permissions:
 
 |===
 
-==== IP Addresses
+==== IPv6
+
+[WARNING]
+====
+Using IPv6 addresses is currently not supported but it is not disabled.
+====
+
+A common practice to disable IPv6 at the node level is to configure the following kernel parameter:
```

Comment:
Where does this need to be configured ? Mention the exact file that needs to be adapted.

Reply:
I didn't want to write a guide on how to disable IPv6 so I just refer to common practices used by sysadmins. IMO we should not encourage disabling IPv6.

Reply:
I mean disabling it at the host level*

Reply:
> IMO we should not encourage disabling IPv6.

Well, customers do disable IPv6 for "security reasons" ;-) and don't care about our recommendations :-/

Reply:
It is a best practice and recommended to turn of all protocols that are not required and not used.
(not only for our systems ;-)) 
That is the reason why at least consulting recommends to turn of IPv6. 

As we do not support IPv6 with CaaSP / K8S at the moment and AFAIK K8S does not support dual-stack - I believe the default should be not to "depend" on it. 

Once we have IPv6 support this should be changed or at least the decision needs to be done during the deployment with Skuba.

Reply:
Dual stack is in alpha in `1.16`.

We can just turn this into "enabling IPv6" if we go with disabling by default in skuba. No strong opinions on this.

Reply:
Can´t we enhance the cilium pod to autodetect if ipv6 is available?
(sounds a bit like working around a missing feature in that pod ;-))